tests/functional/test_api.py

```diff
@@ -6,73 +6,46 @@
 from ghmirror.app import APP
 from ghmirror.data_structures.monostate import UsersCache
 
+class MockResponse:
+    def __init__(self, content, headers, status_code, user):
```

Comment:
`user=''` here (or, better, `user=None`) will avoid all the calls passing in empty user.

Reply:
done